### ISSUE TITLE
Happy path E2E tests failed on 'Run debug and check application stop in the breakpoint' step

### ISSUE THREAD

**Issue by dmytro-ndp** [eclipse-che/che#18034]
### Describe the bug
Happy path E2E tests failed on '_Run debug and check application stop in the breakpoint_' step on minikube and minishift  from time to time:
https://ci.centos.org/view/Devtools/job/devtools-che-nightly-happy-path-test/237/console
https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/basic-MultiUser-Che-check-e2e-tests-against-k8s/2318/consoleFull
```
          ▼ Editor.selectTab "WelcomeController.java"
          ▼ Editor.waitTab "WelcomeController.java"
          ▼ Editor.clickOnTab "WelcomeController.java"
          ▼ Editor.waitTabFocused "WelcomeController.java"
          ▼ TopMenu.selectOption "View"
          ▼ TopMenu.clickOnTopMenuButton "View"
          ▼ Ide.closeAllNotifications
          ▼ NotificationCenter.open
          ▼ NotificationCenter.clickIconOnStatusBar
          ▼ NotificationCenter.waitOpening
          ▼ NotificationCenter.closeAll
          ▼ NotificationCenter.clickCloseAllNotificationsButton
          ▼ NotificationCenter.waitClearNotificationsList
          ▼ TopMenu.clickOnSubmenuItem "Debug"
          ▼ Ide.waitLeftToolbarButton
          ▼ DebugView.clickOnDebugConfigurationDropDown
          ▼ DebugView.clickOnDebugConfigurationItem "Debug (Attach) - Remote"
          ▼ Ide.performKeyCombination ""
          ▼ DebugView.clickOnRunDebugButton
          ▼ Waiting for debugger to connect (threads to appear in "Threads" view)
...
          ▼ Waiting for debugger to connect (threads to appear in "Threads" view)
    1) Run debug and check application stop in the breakpoint


  21 passing (11m)
  1 pending
  1 failing

  1) Validation of debug functionality
       Run debug and check application stop in the breakpoint:
     TimeoutError: Wait timed out after 30001ms
      at /tmp/e2e/node_modules/selenium-webdriver/lib/promise.js:2201:17
      at ManagedPromise.invokeCallback_ (/tmp/e2e/node_modules/selenium-webdriver/lib/promise.js:1376:14)
      at TaskQueue.execute_ (/tmp/e2e/node_modules/selenium-webdriver/lib/promise.js:3084:14)
      at TaskQueue.executeNext_ (/tmp/e2e/node_modules/selenium-webdriver/lib/promise.js:3067:27)
      at asyncRun (/tmp/e2e/node_modules/selenium-webdriver/lib/promise.js:2927:27)
      at /tmp/e2e/node_modules/selenium-webdriver/lib/promise.js:668:7
      at <anonymous>
      at process._tickCallback (internal/process/next_tick.js:188:7)
  From: Task: <anonymous wait>
      at scheduleWait (/tmp/e2e/node_modules/selenium-webdriver/lib/promise.js:2188:20)
      at ControlFlow.wait (/tmp/e2e/node_modules/selenium-webdriver/lib/promise.js:2517:12)
      at thenableWebDriverProxy.wait (/tmp/e2e/node_modules/selenium-webdriver/lib/webdriver.js:934:29)
      at DebugView.waitForDebuggerToConnect (/tmp/e2e/pageobjects/ide/DebugView.ts:53:45)
      at Context.test (/tmp/e2e/tests/e2e_happy_path/HappyPath.spec.ts:259:25)
      at <anonymous>
      at process._tickCallback (internal/process/next_tick.js:188:7)
```

### Che version

 - [ ] latest
 - [x] nightly
 - [ ] other: please specify

### Steps to reproduce
[Happy path tests script](https://github.com/eclipse/che/blob/60566a14a5c76b4d1b842daf17a9bee1e74b2f2a/tests/e2e/tests/e2e_happy_path/HappyPath.spec.ts)

### Runtime
   - [ ] kubernetes (include output of `kubectl version`)
   - [ ] Openshift (include output of `oc version`)
   - [x] minikube 1.1.1 (include output of `minikube version` and `kubectl version`)
   - [x] minishift 3.11 (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

### Screenshots
![screenshot-Run_debug_and_check_application_stop_in_the_breakpoint](https://user-images.githubusercontent.com/1197777/94930321-e794e100-04ce-11eb-8e47-c43250657203.png)

### Screencast
https://youtu.be/jqfECJXL5EQ

### Installation method
   - [x] chectl
     * provide a full command that was used to deploy Eclipse Che (including the output)
     * provide an output of `chectl version` command
   - [ ] OperatorHub
   - [ ] I don't know

### Environment
   - [ ] my computer
       - [ ] Windows
       - [ ] Linux
       - [ ] macOS
   - [ ] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify)
   - [x] other: CRW CCI, Centos CI

**Comment by sleshchenko** [comment]
Assigning severity/blocker since AFAIU it prevents new PRs from being merged.

**Comment by tsmaeder** [comment]
@dmytro-ndp youtube says "this video is private".

**Comment by dmytro-ndp** [comment]
@tsmaeder: sorry for that.
Now screencast should be public https://youtu.be/jqfECJXL5EQ

**Comment by tsmaeder** [comment]
@dmytro-ndp the sequence of events in the video seems to be:

1. Set a break point
2. Start the application
3. Open the preview
4. Connect the debugger
5. Nothing

I don't see a refresh in the browser: why would you expect the breakpoint to be hit if the code is not executed again?

**Comment by dmytro-ndp** [comment]
@tsmaeder: if you look at normal debug flow https://www.youtube.com/watch?v=dN1oprNuyyg&feature=youtu.be&t=600, you might notice that there should threads in THREADS list be appeared after connection to debugger:

#### Expected THREADS list:
![Screenshot from 2020-10-08 19-13-35](https://user-images.githubusercontent.com/1197777/95485586-a2732200-099a-11eb-8d8a-8f05ac2d1124.png)

#### Actual THREADS list
![94930321-e794e100-04ce-11eb-8e47-c43250657203](https://user-images.githubusercontent.com/1197777/95485561-97b88d00-099a-11eb-9c3e-da3c9b77caec.png)

**Comment by dmytro-ndp** [comment]
Changed priority, because the issue wasn't reproduced during the last few days.

**Comment by rhopp** [comment]
@dmytro-ndp The crucial difference between "normal" flow and this bug is that in the "normal" flow, the preview widget gets reloaded (which triggers the breakpoint). That's what's missing in the screencast of this bug, as Thomas was saying.

**Comment by dmytro-ndp** [comment]
@rhopp: that is not true. If you look at [normal flow screen cast](https://www.youtube.com/watch?v=dN1oprNuyyg&feature=youtu.be&t=600), you will see that threads had appeared in the THREADS list BEFORE the test was refreshed preview tabs.

**Comment by rhopp** [comment]
TBH... I blame that on slow UI. 
I don't see how the breakpoint could have been triggered, when the code was not being executed (the code gets executed, when the page is loading...)

**Comment by rhopp** [comment]
Ok. Dmytro proved me wrong... I was thinking (the same way as Thomas), that the test failed on waiting for the breakpoint to be hit, but that's not the case. 
After executing "Debug (Attach)" in debug view, the test is waiting for active threads to be shown in the left panel in "Threads" section ([here](https://github.com/eclipse/che/blob/master/tests/e2e/tests/e2e_happy_path/HappyPath.spec.ts#L259) and that's what fails, because on the failed cases, all the sections in the debug panel are collapsed.

**Comment by dmytro-ndp** [comment]
@tsmaeder, @l0rd, @rhopp: the issue has being reproduced quite often in 7.21.0-SNAPSHOT, 7.22.0-SNAPHSOT last days:

PR check:
- https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Multiuser-Che-PR-check-E2E-Happy-path-tests-against-k8s/3919/console
- https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Multiuser-Che-PR-check-E2E-Happy-path-tests-against-k8s/3918/console

Nightly tests:
- https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/basic-MultiUser-Che-check-e2e-tests-against-k8s/2633/console 
- https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/basic-MultiUser-Che-check-e2e-tests-against-k8s/2635/console

**Comment by dmytro-ndp** [comment]
The issue severity set to `blocker` because it's quite often reason of PR check failure last weeks.

**Comment by azatsarynnyy** [comment]
It looks like an upstream Theia issue rather than a plugin related.
We'll look at it during the next sprint.

**Comment by l0rd** [comment]
After discussion with QE team we agreed that the proper fix for this issue is to use another indicator to check if the debugger has been attached or not (status bar). This fix will be implemented by QE team.

As mentioned by @azatsarynnyy editors team will look at why the debug view sections get collapsed now. That's not a blocker but still something to investigate.

**Comment by RomanNikitenko** [comment]
Within working on another issue I had failed happy path tests due to the issue.
So I tried to reproduce it.

First I could not reproduce it at all.
Then on a video from  happy path tests report I noticed refreshing a page on the initial starting IDE step.
So next I was able to reproduce it, but only after refreshing a page exactly on initial starting IDE step.

I didn't investigate the problem, so I don't have more details, but maybe refreshing a page on the initial starting IDE step leads to some incorrect state for views.

**Comment by azatsarynnyy** [comment]
Should be fixed by https://github.com/eclipse/che/issues/18564

**Comment by benoitf** [comment]
when using github action to run happy path tests, it's reproduced almost 2 out of 3 runs

**Comment by azatsarynnyy** [comment]
> when using github action to run happy path tests, it's reproduced almost 2 out of 3 runs

Thanks! @vzhukovs is looking at it (within https://github.com/eclipse/che/issues/18564) during the next sprint.

**Comment by dmytro-ndp** [comment]
The issue has been reproduced with Eclipse Che 7.26.0-SNAPSHOT https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/blue/organizations/jenkins/Multiuser-Che-PR-check-E2E-Happy-path-tests-against-k8s/detail/Multiuser-Che-PR-check-E2E-Happy-path-tests-against-k8s/4497/:
```
          ▼ Waiting for debugger to connect (threads to appear in "Threads" view)
    1) Run debug and check application stop in the breakpoint
  [ERROR] CheReporter runner.on.fail: Validation of debug functionality Run debug and check application stop in the breakpoint failed after 34862ms

  21 passing (9m)
  1 pending
  1 failing

  1) Validation of debug functionality
       Run debug and check application stop in the breakpoint:
     TimeoutError: Wait timed out after 30015ms
      at /tmp/e2e/node_modules/selenium-webdriver/lib/promise.js:2201:17
      at ManagedPromise.invokeCallback_ (/tmp/e2e/node_modules/selenium-webdriver/lib/promise.js:1376:14)
      at TaskQueue.execute_ (/tmp/e2e/node_modules/selenium-webdriver/lib/promise.js:3084:14)
      at TaskQueue.executeNext_ (/tmp/e2e/node_modules/selenium-webdriver/lib/promise.js:3067:27)
      at asyncRun (/tmp/e2e/node_modules/selenium-webdriver/lib/promise.js:2927:27)
      at /tmp/e2e/node_modules/selenium-webdriver/lib/promise.js:668:7
      at process._tickCallback (internal/process/next_tick.js:68:7)
  From: Task: <anonymous wait>
      at scheduleWait (/tmp/e2e/node_modules/selenium-webdriver/lib/promise.js:2188:20)
      at ControlFlow.wait (/tmp/e2e/node_modules/selenium-webdriver/lib/promise.js:2517:12)
      at thenableWebDriverProxy.wait (/tmp/e2e/node_modules/selenium-webdriver/lib/webdriver.js:934:29)
      at DebugView.waitForDebuggerToConnect (/tmp/e2e/pageobjects/ide/DebugView.ts:53:45)
      at Context.test (/tmp/e2e/tests/e2e_happy_path/HappyPath.spec.ts:279:25)
      at process._tickCallback (internal/process/next_tick.js:68:7)
```
![screenshot-Run_debug_and_check_application_stop_in_the_breakpoint (2)](https://user-images.githubusercontent.com/1197777/107283945-bb353680-6a65-11eb-962e-3753302792b1.png)

There was error message in [vscode-javahpg.container.log](https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Multiuser-Che-PR-check-E2E-Happy-path-tests-against-k8s/4497/artifact/logs-and-configs/workspace-logs/vscode-javahpg.container.log/*view*/):
```
Debug configuration provider has been registered: java
{
  message: 'Starting Java server with: /usr/lib/jvm/default-jvm/bin/java --add-modules=ALL-SYSTEM --add-opens java.base/java.util=ALL-UNNAMED --add-opens java.base/java.lang=ALL-UNNAMED -Declipse.application=org.eclipse.jdt.ls.core.id1 -Dosgi.bundles.defaultStartLevel=4 -Declipse.product=org.eclipse.jdt.ls.core.product -Dfile.encoding=utf8 -XX:+UseParallelGC -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true -Xmx1G -Xms100m -noverify -jar /tmp/vscode-unpacked/redhat.java.latest.orguqpuxuw.java-0.69.0-2547.vsix/extension/server/plugins/org.eclipse.equinox.launcher_1.5.800.v20200727-1323.jar -configuration /home/theia/home/theia/redhat.java/0.69.0/config_linux -data /home/theia/home/theia/redhat.java/jdt_ws',
  level: 'info',
  timestamp: '2021-02-08 18:35:30.438'
}
{
  message: '[Error - 6:35:42 PM] Feb 8, 2021, 6:35:42 PM Failed to build workspace.\n' +
    'jdt.ls-java-project does not exist\n' +
    'Java Model Exception: Java Model Status [jdt.ls-java-project does not exist]\n' +
    '\tat org.eclipse.jdt.internal.core.JavaElement.newJavaModelException(JavaElement.java:583)\n' +
    '\tat org.eclipse.jdt.internal.core.Openable.generateInfos(Openable.java:256)\n' +
    '\tat org.eclipse.jdt.internal.core.JavaElement.openWhenClosed(JavaElement.java:596)\n' +
    '\tat org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:326)\n' +
    '\tat org.eclipse.jdt.internal.core.JavaElement.getElementInfo(JavaElement.java:312)\n' +
    '\tat org.eclipse.jdt.internal.core.JavaElement.getChildren(JavaElement.java:267)\n' +
    '\tat org.eclipse.jdt.internal.core.JavaProject.getPackageFragmentRoots(JavaProject.java:2322)\n' +
    '\tat org.eclipse.jdt.internal.core.JavaProject.getPackageFragments(JavaProject.java:2346)\n' +
    '\tat org.eclipse.jdt.ls.core.internal.managers.ProjectsManager.cleanupResources(ProjectsManager.java:214)\n' +
    '\tat org.eclipse.jdt.ls.core.internal.handlers.BuildWorkspaceHandler.buildWorkspace(BuildWorkspaceHandler.java:53)\n' +
    '\tat org.eclipse.jdt.ls.core.internal.handlers.JDTLanguageServer.lambda$26(JDTLanguageServer.java:815)\n' +
    '\tat org.eclipse.jdt.ls.core.internal.handlers.JDTLanguageServer.lambda$50(JDTLanguageServer.java:981)\n' +
    '\tat java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:642)\n' +
    '\tat java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:478)\n' +
    '\tat java.base/java.lang.Thread.run(Thread.java:834)\n',
  level: 'info',
  timestamp: '2021-02-08 18:35:42.229'
}
```

[che-theia container log](https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Multiuser-Che-PR-check-E2E-Happy-path-tests-against-k8s/4497/artifact/logs-and-configs/workspace-logs/theia-idef25.container.log/*view*/)
```
2021-02-08 19:45:21.520 root ERROR [hosted-plugin: 290] Conflict at exporting task configurations: {"type":"che","label":"run-debug","command":"java -jar -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005 spring-petclinic-2.4.2.jar --spring.profiles.active=mysql","target":{"workingDir":"/projects/petclinic/target","component":"maven-container"},"problemMatcher":[]} and {"type":"che","label":"run-debug","command":"java -jar -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005 spring-petclinic-2.4.2.jar --spring.profiles.active=mysql","_scope":"","target":{"workingDir":"/projects/petclinic/target","component":"maven-container"},"problemMatcher":[]} The configuration: {"type":"che","label":"run-debug","command":"java -jar -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005 spring-petclinic-2.4.2.jar --spring.profiles.active=mysql","_scope":"","target":{"workingDir":"/projects/petclinic/target","component":"maven-container"},"problemMatcher":[]} is ignored
Conflict at exporting task configurations: {"type":"che","label":"build","command":"mvn clean package | tee build.txt && tail -n 40 /projects/petclinic/build.txt | grep 'BUILD SUCCESS' > /projects/petclinic/result-build.txt","target":{"workingDir":"/projects/petclinic","component":"maven-container"},"problemMatcher":[]} and {"type":"che","label":"build","command":"mvn clean package | tee build.txt && tail -n 40 /projects/petclinic/build.txt | grep 'BUILD SUCCESS' > /projects/petclinic/result-build.txt","_scope":"","target":{"workingDir":"/projects/petclinic","component":"maven-container"},"problemMatcher":[]} The configuration: {"type":"che","label":"build","command":"mvn clean package | tee build.txt && tail -n 40 /projects/petclinic/build.txt | grep 'BUILD SUCCESS' > /projects/petclinic/result-build.txt","_scope":"","target":{"workingDir":"/projects/petclinic","component":"maven-container"},"problemMatcher":[]} is ignored
Conflict at exporting task configurations: {"type":"che","label":"build-file-output","command":"cd /projects/petclinic && mvn package | tee build-output.txt && tail -n 40 /projects/petclinic/build-output.txt | grep 'BUILD SUCCESS' > /projects/petclinic/result-build-output.txt","target":{"component":"maven-container"},"problemMatcher":[]} and {"type":"che","label":"build-file-output","command":"cd /projects/petclinic && mvn package | tee build-output.txt && tail -n 40 /projects/petclinic/build-output.txt | grep 'BUILD SUCCESS' > /projects/petclinic/result-build-output.txt","_scope":"","target":{"component":"maven-container"},"problemMatcher":[]} The configuration: {"type":"che","label":"build-file-output","command":"cd /projects/petclinic && mvn package | tee build-output.txt && tail -n 40 /projects/petclinic/build-output.txt | grep 'BUILD SUCCESS' > /projects/petclinic/result-build-output.txt","_scope":"","target":{"component":"maven-container"},"problemMatcher":[]} is ignored
Conflict at exporting task configurations: {"type":"che","label":"run","command":"java -jar spring-petclinic-2.4.2.jar --spring.profiles.active=mysql","target":{"workingDir":"/projects/petclinic/target","component":"maven-container"},"problemMatcher":[]} and {"type":"che","label":"run","command":"java -jar spring-petclinic-2.4.2.jar --spring.profiles.active=mysql","_scope":"","target":{"workingDir":"/projects/petclinic/target","component":"maven-container"},"problemMatcher":[]} The configuration: {"type":"che","label":"run","command":"java -jar spring-petclinic-2.4.2.jar --spring.profiles.active=mysql","_scope":"","target":{"workingDir":"/projects/petclinic/target","component":"maven-container"},"problemMatcher":[]} is ignored
Conflict at exporting task configurations: {"type":"che","label":"run-with-changes","command":"java -jar spring-petclinic-2.4.2.jar --spring.profiles.active=mysql","target":{"workingDir":"/projects/petclinic/target","component":"maven-container"},"problemMatcher":[]} and {"type":"che","label":"run-with-changes","command":"java -jar spring-petclinic-2.4.2.jar --spring.profiles.active=mysql","_scope":"","target":{"workingDir":"/projects/petclinic/target","component":"maven-container"},"problemMatcher":[]} The configuration: {"type":"che","label":"run-with-changes","command":"java -jar spring-petclinic-2.4.2.jar --spring.profiles.active=mysql","_scope":"","target":{"workingDir":"/projects/petclinic/target","component":"maven-container"},"problemMatcher":[]} is ignored 
2021-02-08 19:45:21.520 root ERROR Conflict at exporting task configurations: {"type":"che","label":"run-debug","command":"java -jar -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005 spring-petclinic-2.4.2.jar --spring.profiles.active=mysql","target":{"workingDir":"/projects/petclinic/target","component":"maven-container"},"problemMatcher":[]} and {"type":"che","label":"run-debug","command":"java -jar -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005 spring-petclinic-2.4.2.jar --spring.profiles.active=mysql","_scope":"","target":{"workingDir":"/projects/petclinic/target","component":"maven-container"},"problemMatcher":[]} The configuration: {"type":"che","label":"run-debug","command":"java -jar -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005 spring-petclinic-2.4.2.jar --spring.profiles.active=mysql","_scope":"","target":{"workingDir":"/projects/petclinic/target","component":"maven-container"},"problemMatcher":[]} is ignored
Conflict at exporting task configurations: {"type":"che","label":"build","command":"mvn clean package | tee build.txt && tail -n 40 /projects/petclinic/build.txt | grep 'BUILD SUCCESS' > /projects/petclinic/result-build.txt","target":{"workingDir":"/projects/petclinic","component":"maven-container"},"problemMatcher":[]} and {"type":"che","label":"build","command":"mvn clean package | tee build.txt && tail -n 40 /projects/petclinic/build.txt | grep 'BUILD SUCCESS' > /projects/petclinic/result-build.txt","_scope":"","target":{"workingDir":"/projects/petclinic","component":"maven-container"},"problemMatcher":[]} The configuration: {"type":"che","label":"build","command":"mvn clean package | tee build.txt && tail -n 40 /projects/petclinic/build.txt | grep 'BUILD SUCCESS' > /projects/petclinic/result-build.txt","_scope":"","target":{"workingDir":"/projects/petclinic","component":"maven-container"},"problemMatcher":[]} is ignored
Conflict at exporting task configurations: {"type":"che","label":"build-file-output","command":"cd /projects/petclinic && mvn package | tee build-output.txt && tail -n 40 /projects/petclinic/build-output.txt | grep 'BUILD SUCCESS' > /projects/petclinic/result-build-output.txt","target":{"component":"maven-container"},"problemMatcher":[]} and {"type":"che","label":"build-file-output","command":"cd /projects/petclinic && mvn package | tee build-output.txt && tail -n 40 /projects/petclinic/build-output.txt | grep 'BUILD SUCCESS' > /projects/petclinic/result-build-output.txt","_scope":"","target":{"component":"maven-container"},"problemMatcher":[]} The configuration: {"type":"che","label":"build-file-output","command":"cd /projects/petclinic && mvn package | tee build-output.txt && tail -n 40 /projects/petclinic/build-output.txt | grep 'BUILD SUCCESS' > /projects/petclinic/result-build-output.txt","_scope":"","target":{"component":"maven-container"},"problemMatcher":[]} is ignored
Conflict at exporting task configurations: {"type":"che","label":"run","command":"java -jar spring-petclinic-2.4.2.jar --spring.profiles.active=mysql","target":{"workingDir":"/projects/petclinic/target","component":"maven-container"},"problemMatcher":[]} and {"type":"che","label":"run","command":"java -jar spring-petclinic-2.4.2.jar --spring.profiles.active=mysql","_scope":"","target":{"workingDir":"/projects/petclinic/target","component":"maven-container"},"problemMatcher":[]} The configuration: {"type":"che","label":"run","command":"java -jar spring-petclinic-2.4.2.jar --spring.profiles.active=mysql","_scope":"","target":{"workingDir":"/projects/petclinic/target","component":"maven-container"},"problemMatcher":[]} is ignored
Conflict at exporting task configurations: {"type":"che","label":"run-with-changes","command":"java -jar spring-petclinic-2.4.2.jar --spring.profiles.active=mysql","target":{"workingDir":"/projects/petclinic/target","component":"maven-container"},"problemMatcher":[]} and {"type":"che","label":"run-with-changes","command":"java -jar spring-petclinic-2.4.2.jar --spring.profiles.active=mysql","_scope":"","target":{"workingDir":"/projects/petclinic/target","component":"maven-container"},"problemMatcher":[]} The configuration: {"type":"che","label":"run-with-changes","command":"java -jar spring-petclinic-2.4.2.jar --spring.profiles.active=mysql","_scope":"","target":{"workingDir":"/projects/petclinic/target","component":"maven-container"},"problemMatcher":[]} is ignored
```

**Comment by azatsarynnyy** [comment]
If the problem is related to the collapsed views, it's already fixed in https://github.com/eclipse/che/issues/18564 by the PR https://github.com/eclipse/che-theia/pull/970
Try to use Che-Theia `next`.

**Comment by dmytro-ndp** [comment]
@azatsarynnyy: doesn't nightly Eclipse Che use `Che-Theia` next?

The issue has been recently introduced in _Eclipse Che 7.26.0-SNAPSHOT_ nightly E2E tests as well https://main-jenkins-csb-crwqe.apps.ocp4.prod.psi.redhat.com/job/Che/job/e2e/job/minikube/job/basic/job/eclipse-che/528/console

**Comment by azatsarynnyy** [comment]
> @azatsarynnyy: doesn't nightly Eclipse Che use Che-Theia `next`?

It depends on what is specified in devfile. If omitted then  `next` should be used https://github.com/eclipse/che/blob/3ad29ddbf85730bc34ce9407491570f59051389c/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties#L650.

**Comment by azatsarynnyy** [comment]
> The issue has been recently reproduced in _Eclipse Che 7.26.0-SNAPSHOT_ nightly E2E tests as well https://main-jenkins-csb-crwqe.apps.ocp4.prod.psi.redhat.com/job/Che/job/e2e/job/minikube/job/basic/job/eclipse-che/528/console

Thanks @dmytro-ndp for reporting it.
I'm able to reproduce the issue related to the collapsed panels.
I've reopened https://github.com/eclipse/che/issues/18564#issuecomment-776445000, so @vzhukovs will look at it.

**Comment by dmytro-ndp** [comment]
@azatsarynnyy: thanks for investigation!
Actually, Happy path test workspace is using editor `eclipse/che-theia/next`: https://github.com/eclipse/che/blob/master/tests/e2e/files/happy-path/happy-path-workspace.yaml#L12

**Comment by vzhukovs** [comment]
@dmytro-ndp, Vanilla Theia has the same behavior: https://github.com/eclipse/che/issues/18564#issuecomment-776656159, so, what can I propose is to not stick to the specific items in a list, but check, for example whether the view is collapsed or not. If collapsed, then expand. If you will switch to the debugger panel and will refresh the page, the application state will be restored in the same way as it was before reloading. cc @azatsarynnyy

**Comment by azatsarynnyy** [comment]
Thanks @vzhukovs for checking it.
I'm removing `che-theia` label as, Che-Theia state restoring has been already improved by https://github.com/eclipse/che-theia/pull/970

**Comment by dmytro-ndp** [comment]
[crw-ci-test]

**Comment by benoitf** [comment]
Still reproducing on https://github.com/eclipse/che-theia/runs/1890254370?check_suite_focus=true

**Comment by dmytro-ndp** [comment]
@benoitf: thank you update!
I haven't seen the issue in PR check on CRW CCI since resolution.
Could you, please, confirm that the issue is still reproducing in newer Che versions?
If yes, there is too little information in test logs to understand what exactly was going wrong at the moment, and it's hard to find a root cause of issue without screenshot/screencast. Setting TRACE log level should bring more information:
```
-e TS_SELENIUM_LOG_LEVEL="TRACE"
```

**Comment by nickboldt** [comment]
As this has now had 3 milestone designations (7.26, 7.27, 7.28) I've moved it to 7.x as it's either not getting done, or needs an owner to set a valid fixversion.

**Comment by dmytro-ndp** [comment]
Closing the issue because the problem hasn't been reproduced for a long time.
